index.js

```diff
@@ -37,11 +37,27 @@ function handleError(error) {
   process.exitCode = 1;
 }
 
+// The following code block overrides commander's outputHelp()
+delete program.outputHelp;
+program.outputHelp = function (cb) {
+  printHeader();
+  if (!cb) {
+    // eslint-disable-next-line no-param-reassign
+    cb = function (passing) {
+      return passing;
+    };
+  }
+  process.stdout.write(cb(this.helpInformation()));
```

Comment:
I was wondering whether it would be even better if we just shadow the original method, and utilise it in our new method. This way, we do not have to duplicate the original method's logic at all, in case there's an update to the commander's logic when we move to a new version:

```js
program.defaultOutputHelp = program.outputHelp;
program.outputHelp = function (cb) {
  printHeader();
  this.defaultOutputHelp(cb);
};
```